client.go

```diff
@@ -443,6 +489,76 @@ func (client *Client) CfInfo(key string) (map[string]int64, error) {
 	return ParseInfoReply(redis.Values(conn.Do("CF.INFO", key)))
 }
 
+// TdCreate - Allocate the memory and initialize the t-digest
+func (client *Client) TdCreate(key string, compression int64) (string, error) {
```

Comment:
Suggest to change signature from `TdCreate(key string, compression int64) (string, error)` to `TdCreate(key string, compression int64) (bool, error)`
Given that the commands replies with:

```
 OK on success, error otherwise 
 ```
 https://oss.redislabs.com/redisbloom/master/TDigest_Commands/#tdigestcreate

Reply:
Unfortunately, I can't use it here. I tried, but the `redis.String` method can't return bool, and the `redis.Bool` method can't parse "OK" to bool (L496) ...
Probably this is the reason why most of the functions here return (string, error).